src/main.c

```diff
@@ -156,6 +156,10 @@ typedef struct {
 static Model model;
 static Model *g = &model;
 
+/**This will be used to set height we need instantly to fly high.*/
```

Comment:
This comment is kind of hard to understand and the viewer would probably have to read twice. So a better explanation would be "This will be used to allow the player character the ability to instantly fly high, via a height value".

Reply:
change done.